data/json/items/ranged.json

```diff
@@ -2888,10 +2887,10 @@
     "recoil": 120,
     "durability": 8,
     "burst": 15,
-    "clip_size": 45,
-    "reload": 600,
+    "reload": 100,
     "loudness" : 150,
-    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "magazine", 2 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
+    "magazines" : [ "stanag50", "stanag30", "stanag10" ],
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
```

Comment:
Seeing as this weapon now has a magazine implementation. The description of the weapon should be changed. 
> A H&K416 carbine outfitted with a heavier barrel and an extended magazine...